lib/core/text/abstract_text.nit

```diff
@@ -1095,6 +1095,13 @@ abstract class Text
 		end
 	end
 
+	# Copies the content of `self` to `dst`
+	#
+	# Easier, more user-friendly version of `copy_to_native`
+	#
+	# NOTE: `dst` must be large enough to withold `self.bytelen` bytes
+	fun copy_to(dst: NativeString) do copy_to_native(dst, bytelen, 0, 0)
```

Comment:
I have no problem with the idea but I do not like the name. I think it is quite dangerous (not POLA etc) to expose a low-level service (everything with NativeString is low-level) in a high-level class (Text) with a very generic name (copy_to).